backend/src/models/User.ts

```diff
@@ -49,6 +49,9 @@ class User extends Model<User> {
   })
   @Column
   destinationCountry: string;
+
+  @Column
+  createdAt: string;
```

Comment:
This column is provided actually by sequelize by default. I believe you can omit this.